erpnext/accounts/doctype/tax_withholding_group/test_tax_withholding_group.py

```
@@ -0,0 +1,20 @@
+
+# import frappe
+from frappe.tests import IntegrationTestCase
+
+# On IntegrationTestCase, the doctype test records and all
+# link-field test record dependencies are recursively loaded
+# Use these module variables to add/remove to/from that list
+EXTRA_TEST_RECORD_DEPENDENCIES = []  # eg. ["User"]
+IGNORE_TEST_RECORD_DEPENDENCIES = []  # eg. ["User"]
+
+
+class IntegrationTestTaxWithholdingGroup(IntegrationTestCase):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)